Breadcrumbs.py

```diff
@@ -1,12 +1,18 @@
 # -*- coding: utf-8 -*-
-import sublime, sublime_plugin
 import math
+import re
+import html
+
+import sublime
+import sublime_plugin
 
 try:
   xrange
 except NameError:
   xrange = range
 
+settings = sublime.load_settings('Breadcrumbs.sublime-settings')
```

Comment:
> if we could somehow have different regexps for different file types

It should be possible to have an list of scope, regex combinations. We should be able to get the scope at the current cursor position and then select the regex for that, or fall back to the default if it isn't defined. I think that's easier to manage that trying to hook into syntax specific settings. I'm not sure I'm up for it at this point though, too many aspects I have no experience with. 

>  looks like if you call load_settings "too soon", then the settings are empty.

Ah, that would explain a problem I had just now that disappeared on its own. We shouldn't have to read the settings so often, instead we can subscribe to changes. I'll have to look into this, there has to be an example somewhere that deals with settings efficiently.